aten/src/ATen/WrapDimUtils.h

```diff
@@ -5,6 +5,10 @@
 
 namespace at {
 
+static inline int64_t maybe_wrap_dim(int64_t dim, int64_t dim_post_expr, bool wrap_scalar=true) {
```

Comment:
why it can't be just `using` ?

Reply:
that wouldn't work with adding the other overloads below.